packages/vuetify/src/components/VDialog/VDialog.js

```diff
@@ -188,7 +188,7 @@ export default {
       if (e.keyCode === keyCodes.esc && !this.getOpenDependents().length) {
         if (!this.persistent) {
           this.isActive = false
-          const activator = this.getActivator()
+          const activator = this.getActivator(e)
```

Comment:
You can just remove the warning instead, this will try to focus the dialog again once it's closed.

Reply:
fixed 6963943